src/slack_hooks/service/travis.clj

```diff
@@ -77,8 +80,9 @@
 
 
 (defn travis [request]
-  (prn (-> request :params :payload))
-  (slack/notify {:username travis-username
-                 :icon_url travis-avatar
+  ; (prn (-> request :params :payload))
```

Comment:
Want to just kill this and any other `(prn)` or at least hide them behind an `ENV` setting?